en/debug/sensor_uorb_topic_debugging.md

```diff
@@ -0,0 +1,17 @@
+# Sensor/Topic Debugging using the Listener Command
+
+The uORB is an asynchronous `publish()` / `subscribe()` messaging API used for
+inter-thread/inter-process communication. The `listener` command can be used from the *QGroundControl MAVLink Console* to inspect topic (message) values, including the current values published by sensors.
+
+> **Tip** This is a powerful debugging tool because it can be used even when QGC is connected over a wireless link (e.g. when the vehicle is flying).
+
+<span></span>
```

Comment:
Yes. This allows two "notes" to be rendered following immediately after each other. Without this (or something similar) there will always be at least one full line/para between notes.